microservices/case-service/src/main/java/org/entando/bundle/CaseServiceApplication.java

```diff
@@ -1,13 +1,33 @@
 package org.entando.bundle;
 
+import org.camunda.bpm.engine.ProcessEngine;
+import org.camunda.bpm.engine.RuntimeService;
+import org.camunda.bpm.engine.impl.ProcessEngineImpl;
+import org.camunda.bpm.engine.runtime.ProcessInstance;
+import org.camunda.bpm.engine.spring.components.jobexecutor.SpringJobExecutor;
+import org.camunda.bpm.engine.test.Deployment;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+import javax.annotation.PostConstruct;
 
 @SpringBootApplication
 public class CaseServiceApplication {
 
+	@Autowired
+	private ProcessEngine processEngine;
+
 	public static void main(String[] args) {
 		SpringApplication.run(CaseServiceApplication.class, args);
 	}
 
+	@Bean(name = "jobExecutor")
```

Comment:
I think this should be done in `CamundaConfiguration`, not here

Reply:
Done